workshop/exercise-4/README.md

```diff
@@ -1,108 +1,96 @@
-## Clone the Application Repo and Deploy our Application using Kubectl and yaml
+## Deploy vnext Version and Apply Traffic Shifting
 
-### Clone the application repo
-Let's get the code we'll use for today's lab. This repository contains the code for the Fibonacci application as well as various .yaml files we'll use throughout the lab.
+Did you notice that the Fibonacci sequence started with 1? Many would argue that the sequence should actually start with 0. There's a vnext version of the application that starts the sequence with 0 instead of 1. This container image has been built and deployed to dockerhub, and tagged as vnext. We'll deploy that as v2 of our app, and then route a small percentage of the traffic to it.
 
-1. Clone the git repository:
+![](https://github.com/knative/serving/raw/master/docs/spec/images/fibknativev2.png)
```

Comment:
URL is off

Reply:
URL won't work until PR is merged since it's a new image.

Reply:
should work now -- will confirm once we merge.